src/AwareTrait/LoggerAwareTrait.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\AwareTrait;
+
+use Psr\Log\LoggerInterface;
+
+trait LoggerAwareTrait
```

Comment:
Can't we use https://github.com/php-fig/log/blob/master/src/LoggerAwareTrait.php?

Reply:
Because i can't use `psr/log` which suggests the type-hint and psalm throws me error.

Reply:
@vjik would you please take a look at it?

Reply:
> @vjik would you please take a look at it?

Remember our version of `psr/log` its https://github.com/yiisoft/log/blob/dd14613d098584fc14391f74f83aca0be42b232e/composer.json#L23